### PR TITLE
Lucene++: update to 3.0.8.

### PR DESCRIPTION
**Pull request by Gottox**

### CODE REVIEW

**Comment by Gottox** [comment]
musl should be easy to fix. For the i686 tests, I can't tell.

**Comment by ericonr** [comment]
I have #29983, but this looks more complete. Assuming the dependencies can build with this new version, should also be ready.

File: srcpkgs/Lucene++/template.orig [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Template file for 'Lucene++'
```
Comment (ericonr) [inline srcpkgs/Lucene++/template.orig:1]: Leftover file

Comment (Gottox) [reply srcpkgs/Lucene++/template.orig:1]: whoops, thanks

**Pull request closed without merging**